src/biocommons/seqrepo/_internal/translate.py

```diff
@@ -15,9 +15,10 @@
 
 import copy
 import datetime
+from typing import Iterable, Iterator, Optional
 
 
-def translate_db2api(namespace, alias):
+def translate_db2api(namespace: str, alias: str) -> list[tuple[str, Optional[str]]]:
```

Comment:
Might be good to create an enum for namespace in a separate issue (if it doesn't already exist)

Reply:
Maybe -- I think that makes usage of the `store` function complicated because users might want to add variations under arbitrary/custom namespaces

Reply:
That makes sense. Maybe just an enum for the sources that are used frequently in seqrepo?

Reply:
Yeah, or maybe a predefined union of constants? otherwise you have to annotate everything as `namespace_enum | str` and handle both kinds of types every time